creator-node/src/components/contentBlacklist/contentBlacklistController.js

```diff
@@ -83,30 +109,34 @@ const contentBlacklistRemoveController = async (req) => {
  * @param {object} queryParams
  */
 function parseQueryParams (queryParams) {
-  let { id, type, timestamp, signature } = queryParams
+  let { ids, type, timestamp, signature } = queryParams
 
-  if (!id || !type || !timestamp || !signature) {
-    throw new Error('Missing query params: [id, type, timestamp, signature]')
+  if (!ids || ids.length === 0 || !type || !timestamp || !signature) {
```

Comment:
this should also have an explicity Array.isArray check here. the problem is strings also have a .length property that could pass this